specification/web/resource-manager/Microsoft.DomainRegistration/stable/2024-11-01/Domains.json

```diff
@@ -0,0 +1,1616 @@
+            "schema": {
+              "$ref": "#/definitions/Domain"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update App Service Domain": {
+            "$ref": "./examples/UpdateAppServiceDomain.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DomainRegistration/domains/{domainName}/domainOwnershipIdentifiers": {
+      "get": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Lists domain ownership identifiers.",
+        "description": "Description for Lists domain ownership identifiers.",
+        "operationId": "Domains_ListOwnershipIdentifiers",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DomainOwnershipIdentifierCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Domain Ownership Identifiers": {
+            "$ref": "./examples/ListDomainOwnershipIdentifiers.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DomainRegistration/domains/{domainName}/domainOwnershipIdentifiers/{name}": {
+      "get": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Get ownership identifier for domain",
+        "description": "Description for Get ownership identifier for domain",
+        "operationId": "Domains_GetOwnershipIdentifier",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "name",
```

Comment:
>name [](http://example.com/codeflow?start=21&length=4)

Use a more specific name than name, e.g. identifierName. Feedback applies to other methods as well. Consider factoring this parameter out to the parameters section.